src/components/SamplePanel/index.js

```diff
@@ -140,15 +148,29 @@ const SamplePanel = ({sampleName, sampleData, config, reference, socket, panelEx
         }
         />
     )*/
+    mutationsTree: (
+      <MutationsTree
+       data={sampleVariant}
+       renderProp={ showSinglePanel === "mutationsTree" ?
+            (<ContractChart handleClick={() => goToChart(false)}/>) :
+            (<ExpandChart handleClick={() => goToChart("mutationsTree")}/>)
+        }
+      />
+    )
 
   };
 
   /* ---------------   WHAT CHARTS DO WE RENDER?   -------------- */
   const renderCharts = () => {
+  
     if (!panelExpanded) return null;
-    const chartsToShow = showSinglePanel ?
-    charts[showSinglePanel] :
-    [charts.coverage, charts.readLength, charts.coverageOverTime, charts.refSimilarity];
+    var chartsToShow =[];
```

Comment:
this is misleading that we first assign an array to it, then, later in ternary operator it can be wihout array.

Also, do not use `var`, prefer `const` or `let` if it needs to be mutable